helpers/common.go

```diff
@@ -0,0 +1,39 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"net/http"
+)
+
+// Used for unit testing. Allows for redirecting to stubbed functions to assert correct behaviour.
+var (
+	callReadAll = ioutil.ReadAll
+)
+
+// Helper contains a list of all common functions.
+type Helper interface {
+	GetDataFromRequest(r *http.Request) (string, error)
+}
+
+// Impl directly implements the Helper interface.
+type Impl struct {
+}
+
+// NewHelper Returns an Impl.
+func NewHelper() Impl {
+	return Impl{}
+}
+
+// GetDataFromRequest will try to retrieve the Body from a given request and convert it into a string.
+func (h Impl) GetDataFromRequest(r *http.Request) (string, error) {
+
+	// Retrieve the request body.
+	data, err := callReadAll(r.Body)
+	if err != nil {
```

Comment:
It would be good to log the error here instead of just sending it back

Reply:
Added logging here